### DISCUSSION TITLE
automatisch Installation und via ssh script laufen nicht durch

### DISCUSSION THREAD

**Discussion by MiThie13** [camueller/SmartApplianceEnabler#586] [Installation und Betrieb]
Hi,

ich habe zuerst mehrfach die automatische Installation auf meinem Raspberry PI Model 4 B versucht. Sowohl über MacOS als auch Windows alle Dateien herunter geladen.

Ich komme immer bis zu dem Punkt wo der USB Stick wieder herausgezogen wird. Dann leuchte die rote LED dauerhaft und die grüne nur für ca. 20 sek. selbst nach mehreren Stunden ist weder der Port 8080 noch der 10000 erreichbar.

Ich habe dann die Methode via script versucht. Also mittels ssh über Terminal eingeloggt und die beiden Kommandozeilen eingegeben.

Die Installation startet auch, aber bleibt wieder nach ca. 20 sek hängen und startet einen Timer 3600. Dann passiert nichts mehr.

Hier die Ansicht aus dem Terminal was da passiert. Eine /tmp/install.log habe ich nicht gefunden

[Gesicherte Terminal-Ausgabe.txt](https://github.com/camueller/SmartApplianceEnabler/files/13947637/Gesicherte.Terminal-Ausgabe.txt)

Was mache ich falsch?

LG Michael

**Comment by MiThie13** [comment]
Hat sich gerade erledigt. Warum auch immer komme ich jetzt auf den SAE als auch auf Webmin.

Vielen Dank für das Tolle Projekt!!!

**Reply by ntsimpiridis** [reply]
Hallo,

genau das gleiche Problem habe ich auch.
Haben sie eventuell einen Tipp was helfen könnte?
Meine Terminal-Ausgabe deckt sich auch mit ihrer.

Viele Grüße
Niko
